src/main/resources/static/index.html

```diff
@@ -1,10 +1,28 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Shared rent</title>
-</head>
-<body>
-In development...
-</body>
-</html>
+<html lang="pl">
+	<head>
+		<meta charset="utf-8">
+		<title>Shared Rent</title>
+		<link rel="stylesheet" href="css/normalize.css">
+		<link rel="stylesheet" href="css/style.css">
+	</head>
+	<body>
+		<div class="login-page">
+		  <div class="form">
+		    <div class="login-form">
+		      <input type="text" placeholder="username"/>
+		      <input type="password" placeholder="password"/>
+		      <button class="normal-button" onclick="login()">login</button>
+		      <button class="facebook-button">login with facebook</button>
+		      <p class="message">Not registered? <a href="register.html">Create an account</a></p>
+		    </div>
+		  </div>
+		</div>
+		<script>
+			function login() {
```

Comment:
extract it to some .js file and wrap function in window.onload 
https://developer.mozilla.org/pl/docs/Web/API/Window/onload
https://www.w3schools.com/js/js_whereto.asp
(<script src="myScript.js"></script>)